sktime_dl/deeplearning/tests/test_regressors.py

```diff
@@ -47,7 +47,8 @@ def test_regressor_forecasting(
 
     # load univariate time series data
     y = load_airline()
```

Comment:
in sktime, there's also [`make_forecasting_problem()`](https://github.com/alan-turing-institute/sktime/blob/c20e15210682c9f7ed9ae12a15cd60d82160947e/sktime/utils/_testing/forecasting.py#L117) where you can set the number of data points if that helps speeding up the tests, but likely to make convergence more problematic